lxc/config.go

```diff
@@ -1246,3 +1348,53 @@ func (c *configCmd) doContainerMetadataEdit(client lxd.ContainerServer, name str
 
 	return nil
 }
+
+func (c *configCmd) doContainerTemplateFileCreate(client lxd.ContainerServer, containerName string, templateName string) error {
+	return client.CreateContainerTemplateFile(containerName, templateName)
+}
+
+func (c *configCmd) doContainerTemplateFileEdit(client lxd.ContainerServer, containerName string, templateName string) error {
+	if !termios.IsTerminal(int(syscall.Stdin)) {
+		return client.UpdateContainerTemplateFile(containerName, templateName, os.Stdin)
+	}
+
+	reader, err := client.GetContainerTemplateFile(containerName, templateName)
+	if err != nil {
+		return err
+	}
+	content, err := ioutil.ReadAll(reader)
+	if err != nil {
+		return err
+	}
+
+	// Spawn the editor
+	content, err = shared.TextEditor("", content)
+	if err != nil {
+		return err
+	}
+
+	for {
+		reader := bytes.NewReader(content)
+		err := client.UpdateContainerTemplateFile(containerName, templateName, reader)
+		// Respawn the editor
+		if err != nil {
+			fmt.Fprintf(os.Stderr, i18n.G("Error updating template file: %s")+"\n", err)
+			fmt.Println(i18n.G("Press enter to start the editor again"))
```

Comment:
Then yes, It is, in cases where the editor is spawned (text is not read from stdin)